tests/src/Kernel/AlterableSchemaTest.php

```diff
@@ -156,9 +149,8 @@ protected function mockSchema($id, $schema, array $extensions = []): void {
     $extensions['graphql_alterable_schema_test']->expects(static::any())
       ->method('getBaseDefinition')
       ->willReturn('');
-
     // Different extension definition for different tests.
-    switch ($this->getName()) {
+    switch ($this->toString()) {
```

Comment:
wrong conversion, this needs to be `->name()`. Then you also need to revert the changes to testEmptySchemaExtensionAlteredQueryResultPropertyAdded() above.

Reply:
@klausi I think this was changed because of following reason:

It throws an error with Drupal 11, because the `getName()` method exists until [PHPUnit 9.5](https://github.com/sebastianbergmann/phpunit/blob/9.6/src/Framework/TestCase.php#L1035), but it doesn't exists on [PHPUnit 10](https://github.com/sebastianbergmann/phpunit/blob/10.5/src/Framework/TestCase.php).
So, as an alternative, we should use `$this->toString()` method.

@apathak18 Can confirm on this.

Reply:
`toString` is still the wrong method, as it returns something different.

I'm working on the getName fix in https://github.com/drupal-graphql/graphql/pull/1415